dotcom-rendering/src/layouts/LiveLayout.tsx

```diff
@@ -1153,9 +1153,10 @@ export const LiveLayout = (props: WebProps | AppsProps) => {
 							fullWidth={true}
 							data-print-layout="hide"
 							padSides={false}
-							showTopBorder={false}
+							showTopBorder={true}
 							showSideBorders={false}
-							backgroundColour={sourcePalette.neutral[93]}
+							backgroundColour={sourcePalette.neutral[97]}
+							shouldCenter={false}
```

Comment:
We need the top border to cover the width of the page. `shouldCenter` [introduces a max-width](https://github.com/guardian/dotcom-rendering/blob/main/dotcom-rendering/src/lib/center.ts)

Reply:
Does this still work with the full-width ads? There's still an adtest floating around `adtest=fullwidth-banner`